src/index.html

```diff
@@ -3,19 +3,20 @@
   <meta charset="utf-8">
   <title>Hoover</title>
 
-  <link rel="stylesheet" href="bootstrap.css">
-  <link rel="stylesheet" href="font-awesome.min.css">
-  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="/bootstrap.css">
+  <link rel="stylesheet" href="/font-awesome.min.css">
+  <link rel="stylesheet" href="/style.css">
 </head>
 <body>
 
 <div class="container-fluid" id="app">
 </div>
 
-<script src="jquery.min.js"></script>
-<script src="bootstrap.js"></script>
+<script src="/jquery.min.js"></script>
+<script src="/bootstrap.js"></script>
 <script>
-document.write('<script type="text/javascript" src="app.js?t='
+window.HOOVER_UI_CONFIG = {page: 'search'}
```

Comment:
Instead of using a public variable as the settings argument for the `index.js` entry point, why not first include the `index.js` script, then call its entry point with normal calling syntax?

I'm thinking a single function entry point might be easier to understand than setting a heap of global variables (that should somehow be constants) before including the actual script.